src/components/views/elements/JoinRuleDropdown.tsx

```diff
@@ -24,6 +24,7 @@ interface IProps {
     label: string;
     width?: number;
     labelInvite: string;
+    labelKnock: string;
```

Comment:
This should be the optional:

```suggestion
    labelKnock?: string;
```